apps/web/src/pages/SearchRequests/SearchPage/components/SearchContainer.tsx

```diff
@@ -447,6 +455,39 @@ const SearchContainerApi = () => {
     ? candidatesData?.countPoolCandidatesByPool
     : [];
 
+  /**
+   * Announce the candidate count to users in a less verbose way
+   *
+   * Note: 3 is a magic number, our current candidate count is causing
```

Comment:
Not sure I understand what this comment means: what is 3 a magic number for?

Reply:
Okay, skills should now be announced only after searching and I attempted to make the comment clearer.

[wait on skills dirty before announcing, improve comments](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7010/commits/15ddda8ecce523a552917b63d2916c1008f48f44)